src/sentry/static/sentry/app/views/organizationContext.tsx

```diff
@@ -224,29 +250,30 @@ const OrganizationContext = createReactClass({
         });
       }
     });
-  },
+  }
 
   getOrganizationDetailsEndpoint() {
     return `/organizations/${this.getOrganizationSlug()}/`;
-  },
+  }
 
-  getTitle() {
+  getTitle(): string {
```

Comment:
I believe it's fine if you don't add a return type here 😉

Please see:

![image](https://user-images.githubusercontent.com/29228205/99364491-516a1e00-28b6-11eb-9662-7f54d6e19357.png)